### PR TITLE
🚀  Add Pool Preview to Pre-connected Landing Page

### PR DESCRIPTION
**Pull request by chase-manning**
The landing page previously would not show the pool preview until after the user had connected their wallet.  
This is because it requires a web3 connection to load the data.  
This PR introduces an Infura fallback option for the pool preview so that this data can load when a wallet is not connected.  
And changes the pool preview to always display on the landing page.

### CODE REVIEW

**Comment by danhper** [comment]
Task linked: [CU-42qzwt Setup Infura Fallback for Landing Page](https://app.clickup.com/t/42qzwt)

File: src/state/poolsListSlice.ts [hunk 1]
```diff
@@ -71,6 +75,16 @@ export const fetchState =
     dispatch(fetchPositions({ backd }));
   };
 
+export const fetchPreviewState = (): AppThunk => (dispatch) => {
+  const provider = new ethers.providers.InfuraProvider("kovan", INFURA_ID);
+  const backd = createBackd(provider, { chainId: 42 });
```
Comment (danhper) [inline src/state/poolsListSlice.ts:80]: Let's maybe not hard code kovan an chain id 42 here?

**Review by danhper — COMMENTED** [review]
I left a fairly minor comment that can definitely be tackled later!
Otherwise, I think it looks great, thanks!

File: src/state/poolsListSlice.ts (hunk 1 above)
Comment (chase-manning) [reply src/state/poolsListSlice.ts:80]: My thought process here was that this preview state will only be called when the user has not connected their wallet. At that time we don't know what network they are on, and so we will have to display a network by default. I think the best network to display by default is mainnet, but we don't have contracts live there yet to display, so using kovan in the interim.

Do you think we want this preview feature to show on the site while we are still only on Kovan? Showing the Kovan pools? Or do you think it is better to delay this preview until we are live on mainnet and then show the mainnet version?

Comment (chase-manning) [reply src/state/poolsListSlice.ts]: Have changed this to use mainnet!

Also have put this feature behind a `protocolLive` check, which essentially disables it. That way we can merge this PR and just remove the flag when we go live on mainnet. Should be a bit better than keeping the PR around I think. Have added a task for mainnet launch to remove this flag and test the feature again.

**Pull request merged**